tools/lsp/language/semantic_tokens.rs

```diff
@@ -93,6 +93,7 @@ pub fn get_semantic_tokens(
                 SyntaxKind::ChildrenPlaceholder => Some((self::MACRO, 0)),
```

Comment:
Earlier in the DeclaredIdentifier we should also have it as a definition for a local variable.

We also probably need support for hover and goto def in token_info.rs, but this can be done in a follow up.

Maybe add some tests around https://github.com/slint-ui/slint/blob/069313f39ee17b2d681c9ebde3d6a00a7be23afc/tools/lsp/language/completion.rs#L1332 (extending that one or another test) to check that completion works fine (it should)